app/Jobs/ProvisionQueryserviceNamespaceJob.php

```diff
@@ -75,31 +81,33 @@ public function handle()
             ],
         ]);
 
-        $response = curl_exec($curl);
-        $err = curl_error($curl);
-
-        curl_close($curl);
+        $response = $this->request->execute(); 
+        $err = $this->request->error();
 
+        $this->request->close();
         if ($err) {
-            $this->fail(
-                new \RuntimeException('cURL Error #:'.$err)
-            );
-
-            return; //safegaurd
+            $this->fail( new \RuntimeException('cURL Error #:'.$err) );
+            return;
         } else {
             if ($response === 'CREATED: '.$this->namespace) {
                 $qsns = QueryserviceNamespace::create([
                     'namespace' => $this->namespace,
-                    //'internalHost' => $this->internalHost,
+                    //'internalHost' => $this->internalHost, TODO is this required?
```

Comment:
Not right now, we could probably nuke this line.

Reply:
done